### ISSUE TITLE
Add documentation for options

### ISSUE THREAD

**Issue by javierv** [adamcooper/vestal_versions#10]
Hello, Adam.

There's a mention to the :only and :except options at the end of the document, but other options, such as :initial_version, are only mentioned in the source code unless there's a documentation site I haven't found.

It would be nice to have all of them documented in the same place.

Thank you.

**Comment by adamcooper** [comment]
You are absolutely right.  Most people don't read through source to find the options.

Thanks for the notice.  

I'll try and add it in the following your week.
